projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -388,6 +391,19 @@ export class DatagridComponent<R> implements OnInit, AfterViewInit {
         }
         return [];
     }
+    /**
+     * Stream to send push notifications when ever columns are updated by the calling components using this data grid
+     */
+    private columnsUpdated = new Subject<boolean>();
+
+    columnsUpdatedObservable = this.columnsUpdated.asObservable();
```

Comment:
Is this supposed to be `@Output`? You documented a private `Subject` but didn't document the public API? Only use observables if you absolutely must. `EventEmitters` are the standard way to communicate between components, you should always use that when possible.

What is the boolean value passed to the events for?